### PR TITLE
Elasticache cluster public exposure

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/intel/aws/elasticache.py [hunk 1]
```diff
@@ -65,6 +65,10 @@ def load_elasticache_clusters(
             cluster.region = elasticache_cluster.region
         SET cluster.lastupdated = $aws_update_tag
 
+        WITH cluster, elasticache_cluster
+        UNWIND elasticache_cluster.SecurityGroups as sg
+            MERGE (:EC2SecurityGroup{id: sg.SecurityGroupId})<-[:MEMBER_OF_EC2_SECURITY_GROUP]-(cluster)
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/elasticache.py:70]: create separate function for this

File: cartography/intel/aws/elasticache.py [hunk 2]
```diff
@@ -142,6 +146,7 @@ def sync(
             common_job_parameters['pagination']['elasticache']['hasNextPage'] = has_next_page
 
     load_elasticache_clusters(neo4j_session, clusters, current_aws_account_id, update_tag)
+    attach_elasticache_clusters_to_security_groups(neo4j_session, clusters, update_tag)
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/elasticache.py:149]: Method not declared

File: cartography/intel/aws/elasticache.py (hunk 1 above)
Comment (ganeshjagdale9545) [reply cartography/intel/aws/elasticache.py:70]: also create cleanup job for this

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
Minor changes needed

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**